README.adoc

```diff
@@ -149,6 +149,35 @@ Number of Examples: 54 (0 deprecated)
 |===
 // examples: END
 
+== Deploying the examples in openshift or dev-sandbox using devfiles
+
+=== Pre-requisites
+
+- https://odo.dev/docs/overview/installation[Install odo (Preferred odo version - 2.x)]
+
+=== Steps:
+
+- First login to your openshift or dev-sandbox and create a new project. Here $EXAMPLE  is the name of the example you want to deploy:
+
+        $ oc new-project csbex-$EXAMPLE
+
+- To set the specific example you want to deploy as an env variable (SUB_FOLDER):
+
+        $ odo config set --env SUB_FOLDER=$EXAMPLE
```

Comment:
between those two commands, it would be a `odo create` command, right? 
something like 
`odo create csb-ubi8`

Reply:
Oh yeah, i forgot to add it. Thanks!